<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>135</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>135</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>135</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Файл:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Хеш-сумма:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="browseButton">
        <property name="text">
         <string>Обзор...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Алгоритм:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>CRC32</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MD5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SHA1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SHA256</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SHA512</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Keccak_256</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Keccak_512</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QPushButton" name="calcButton">
        <property name="text">
         <string>Посчитать</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QPushButton" name="compareButton">
        <property name="text">
         <string>Сравнить с...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="6">
       <widget class="QLineEdit" name="fileName"/>
      </item>
      <item row="1" column="1" colspan="6">
       <widget class="QLineEdit" name="checksum"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
